test/test_biweight.jl

```diff
@@ -0,0 +1,73 @@
+function _test_cov_vec(ce::CovarianceEstimator, X::AbstractMatrix{<:Real})
+    c = cov(ce, X)
+    @inbounds for i in 1:size(X, 2)
+        @test cov(ce, @view(X[:, i])) ≈ c[i, i] atol = 1e-12 rtol = 1e-16
+    end
+end
+
+function _test_cov_vecvec(ce::CovarianceEstimator, X::AbstractMatrix{<:Real})
+    c = cov(ce, X)
+    @inbounds for i in 1:size(X, 2)
+        for j in 1:size(X, 2)
+            cij = cov(ce, @view(X[:, i]), @view(X[:, j]))
+            @test cij ≈ c[i, j] atol = 1e-12 rtol = 1e-16
+        end
+    end
+end
+
+@testset "BiweightMidcovariance" begin
+    for c in (9, 5)
+        for modify_sample_size in (true, false)
+            @testset "basic properties c=$c, modify_sample_size=$modify_sample_size" begin
+                ce = BiweightMidcovariance(; c, modify_sample_size)
```

Comment:
```suggestion
                ce = BiweightMidcovariance(; c=c, modify_sample_size=modify_sample_size)
```
This new syntax doesn't work on older versions of Julia